firefox-ios/Client/Frontend/Browser/Toolbars/AddressToolbarContainer.swift

```diff
@@ -29,12 +29,7 @@ final class AddressToolbarContainer: UIView,
                                      URLBarViewProtocol {
     private enum UX {
         static let compactLeadingEdgeEditing: CGFloat = 8
-        static let compactLeadingEdgeDisplay: CGFloat = 16
-        static let compactTrailingEdge: CGFloat = 16
-
-        static let regularHorizontalEdgeEditing: CGFloat = 16
-        static let regularHorizontalEdgeWebsite: CGFloat = 16
-        static let regularHorizontalEdgeHomepage: CGFloat = 24
+        static let compactHorizontalEdge: CGFloat = 24
```

Comment:
Figma shows the compact space with 16pt and not 24pt. 24pt is just for regular layout.